tests/account_test.js

```diff
@@ -2,10 +2,20 @@ const r_ship = Math.floor(Math.random() * 5) + 1
 
 Feature('Guestbook');
     
-Scenario('verify username and add entry to guestbook',  ({ I }) => {
+Scenario('verify username',  ({ I }) => {
+    //Given I am on a Janus obituary page
+    //When I visit Virginia's Obituary
     I.amOnPage('/us/obituaries/chicagotribune/name/virginia-gruchalski-obituary?pid=196167379');
+    //Then I should see her name in the heading
     I.see('Virginia','.Heading-sc-4h3nqe-0')
+});
+Scenario('add entry to guestbook',  ({ I }) => {
```

Comment:
Update Scenario to have `I can ...` statement.

Reply:
Yes, this needs a little bit more context, like Gilbert said!